README.md

```diff
@@ -206,15 +206,15 @@ Deploy Rocket.Chat stack to your server with [D2C](https://d2c.io/). Scale with
 [![Deploy](https://github.com/mastappl/images/blob/master/deployTo.png)](https://panel.d2c.io/?import=https://github.com/d2cio/rocketchat-stack/archive/master.zip/)
 
 ## Syncloud.org
-Run Rocket.Chat on your easy to use personal device.
+Run Rocket.Chat on your easy to use a personal device.
```

Comment:
This is wrong. The existing sentence is correct @midopooler

Reply:
can you fix the PR?